.pre-commit-config.yaml

```diff
@@ -24,12 +24,12 @@ repos:
       - id: isort
         additional_dependencies: [toml]
   - repo: https://github.com/asottile/pyupgrade
-    rev: v2.32.1
+    rev: v2.38.2
     hooks:
       - id: pyupgrade
         args: [--py37-plus]
```

Comment:
Just noticed `pyupgrade` is still referencing Python 3.7.  Looks like it'll replace `FsPathMock` in tests if this is updated to `--py38-plus`.

Reply:
IIRC, there's a bunch of stuff that is flagged for Py3.6/3.7 deprecation that we could clean up - mostly related to fsencode/decode and places where Pathlib objects are being used.

Reply:
yeah....we can save it for later.

btw, this is one of the automated change it made...although, it left the imports for flake then to get made at:

```
@@ -194,11 +189,8 @@ def test_unpack_fail(mock_tools, tmp_path):
     wix_zip_path = os.fsdecode(tmp_path / "tools" / "wix.zip")
     # Consider to remove if block when we drop py3.7 support, only keep statements from else.
     # MagicMock below py3.8 doesn't have __fspath__ attribute.
-    if sys.version_info < (3, 8):
-        wix_zip = FsPathMock(wix_zip_path)
-    else:
-        wix_zip = MagicMock()
-        wix_zip.__fspath__.return_value = wix_zip_path
+    wix_zip = MagicMock()
+    wix_zip.__fspath__.return_value = wix_zip_path
 ```